test/scalarstats.jl

```diff
@@ -71,7 +71,7 @@ z2 = [8. 2. 3. 1.; 24. 10. -1. -1.; 20. 12. 1. -2.]
 
 ##### Dispersion
 
-@test span([3, 4, 5, 6, 2]) == (2:6)
+@test StatsBase.span([3, 4, 5, 6, 2]) == (2:6)
```

Comment:
Where was `span` coming from before? Was there some kind of issue with calling it unqualified?